api/src/data/models/action-log-model.ts

```diff
@@ -0,0 +1,24 @@
+export interface ActionLog {
+    id: number;
+    action_id: number; // nullable or not isnt specified
+    hazard_id: number;
+    old_actor_employee_id?: number;
```

Comment:
Note that
```
old_actor_employee_id?: number;
```

is not the equivalent of
```
old_actor_employee_id: number | null;
```

If you easier future upgradeability check out how this would be implemented in Sequelize, and try to match that pattern.